HelpSource/Classes/ServerOptions.schelp

```diff
@@ -118,7 +118,7 @@ method:: verbosity
 Controls the verbosity of server messages. A value of 0 is normal behaviour. -1 suppresses informational messages. -2 suppresses informational and many error messages, as well as messages from Poll. The default is 0.
 
 method:: zeroConf
-A Boolean indication whether or not the server should publish its port using zero configuration networking, to facilitate network interaction. This is true by default; if you find unacceptable delays (beachballing) upon server boot, you can try setting this to false.
+A Boolean indication whether or not the server should publish its port using zero configuration networking, to facilitate network interaction. This is false by default.
```

Comment:
isn't this a breaking change? if so, can it go into a separate PR and then into 3.11?

Reply:
I intended only to correct the documentation. I made this change based off `zeroConf's default value in`ServerOptions` which is `false`. Looking at this more closely, I see that `scsynth` and `supernova` default to `-R 1`. I'm not sure what we should do about this discrepancy. In any case, I'll remove this from the PR.

Reply:
done

Reply:
ok, thanks!